app/views/ansible_credential/edit.html.haml

```diff
@@ -1,2 +1,2 @@
 #main_div
-  = render :partial => 'credential_form', :locals => {:newRecord => false}
```

Comment:
You can also delete the `_credential_form.html.haml` as you're deleting this line.